indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -565,6 +565,16 @@ def _process(self):
         return jsonify_data(html=_render_registration_details(self.registration))
 
 
+class RHRegistrationReset(RHManageRegistrationBase):
+    """Reset a registration back to a non-approved status"""
+
+    def _process(self):
+        self.registration.update_state(approved=False)
+        db.session.flush()
+        logger.info('Registration %s was reset by %s', self.registration, session.user)
```

Comment:
I think logging this action to the event log (`event.log(...)`) would be useful (let me guess, it's missing for all the other actions as well? ;))

Reply:
`update_state()` triggers the `registration_state_updated` signal, which is connected to [`log_registration_updated`](https://github.com/indico/indico/blob/master/indico/modules/events/registration/logging.py#L33) and effectively registers all registration state changes on the event log.

Reply:
forgot about that ;) never mind th en